td-agent/templates/etc/init.d/td-agent.erb

```diff
@@ -0,0 +1,348 @@
+#!/bin/sh
```

Comment:
Please copy td-agent/templates/etc/init.d/td-agent.erb as is from omnibus-td-agent and point the revision before modifying it.

Reply:
Sure. I'll do it.

Reply:
The original erb is imported in https://github.com/clear-code/td-agent-builder/pull/19/commits/a09f3d32b49296616dd72dc502c3a2dd11e05a77.
And modified in https://github.com/clear-code/td-agent-builder/pull/19/commits/dc464af9a3e7fad654c5cb677bd46d6df7324315.